### ISSUE TITLE
Proposal: Searching and information retrieval

### ISSUE THREAD

**Issue by nealhaddaway** [mcguinlu/ESinR#13]
### Chapter/Section contents 

How to plan and conduct searches for information in a synthesis:
- Theory (links to external educational resources on building search strategies, systematic searching etc.)
- Sources (bibliographic databases, repositories, archives, organisations)
- Tools/methods (database APIs, API packages, web-scraping, bespoke tools, manual)
- Reporting (need for a priori design, transparency)

### List of R packages that should be covered in this Chapter/Section?

litsearchr
citationchaser
RSelenium
wosr
rscopus

**Comment by mcguinlu** [comment]
Thanks for suggesting this @nealhaddaway 🚀 This is one of the key areas where I think R can really help ESers, and make the ES process a lot more robust and reproducible (vs e.g. copying and pasting into Ovid).

I noticed you recommended `RSelenium` - I think that, along with things like `rvest`, are great packages to discuss, as webscraping can definitely help make ESers lives easier. One side effect of including webscraping as a practice is that the chapter will need to touch lightly () on good practice when doing so (e.g. forward on a contact email if your script goes wild, incorporate delays between scraping), and signpost towards more detailed discussions of the issue.

I'd be very keen to have input from @elizagrames (as maintainer of litsearchr) and Ciara Keenan [GitHub handle TB] (as info retrieval at Campbell) for this chapter, along with your own as author of `citationchaser`!

I've added a new empty chapter on this topic ("chapters/11_information_retrieval.Rmd") - would you be happy to fork the repo, open a PR and begin to flesh it out?

**Comment by nealhaddaway** [comment]
Oi oi! Just spotted this again and see (wonderfully) that it's still active! Great work! @DrMattG, @kylehamilton and I are refurbishing metaverse into a collection of packages for evidence synthesis in R - maybe we should touch base soon?

**Comment by mcguinlu** [comment]
Yes, absolutely! I'm keen to start making some progress on this (which is why there has been some activity on the repo recently, though as always, not as much as I'd like due to it being limited to spare time 😅)

Sounds v. exciting what you and the guys are up to - would be great to catch-up/help out if I can! I'll ping you from my new email address, just so you have it.

**Comment by nealhaddaway** [comment]
Awesome - of course you should definitely join the metaverse2.0 team if you're at all able?! We've made some really exciting progress so far. Will look out for your email!
________________________________
